en/components/select.md

```diff
@@ -50,7 +42,7 @@ Then all that is needed is to define an `<igx-select>` and several items. We use
 </igx-select>
 ```
 
-Another way to do it would be to use a collection of the items that we want to display with the [ngForOf](https://angular.io/api/common/NgForOf) directive:
+Another way to do it would be to use a collection of the items that we want to display, using the [ngForOf](https://angular.io/api/common/NgForOf) directive:
```

Comment:
```suggestion
Another way to do it would be to use a collection of items that we want to display using the [*ngFor](https://angular.io/api/common/NgForOf) structural directive:
```